WS2012R2/lisa/remote-scripts/ica/Core_LIS_CD.sh

```diff
@@ -97,7 +97,14 @@ if [ $os_VENDOR != "Ubuntu" ] && [ $os_VENDOR != "Debian" ]; then
 fi
 sleep 1
 LogMsg "Mount the CDROM"
-mount /dev/cdrom /mnt/
+for drive in $(ls /dev/cdrom*)
```

Comment:
The test scenario is to see if data can be read from a CD. My opinion is that you can stop after the first CD (/dev/cdrom) is mounted (as in the deleted line 100).